src/cljs/main/broadfcui/common/table/table.cljs

```diff
@@ -145,7 +145,7 @@
                                  {:initial-text (:filter-text query-params)
                                   :on-filter #(swap! state update :query-params assoc :filter-text % :page-number 1)}
                                  (:inner filter-bar-props))]]))
-         (list* (:items toolbar))]
+         (list* ((:items toolbar) {:columns columns}))]
```

Comment:
I don't think this is right--`columns` is the static column definitions. `column-display`?